src/components/Contentful/sass/components/_new_design_overrides.scss

```diff
@@ -179,6 +179,40 @@ ul li:before {
     }
   }
 
+  &.green {
+    background-color: $mt-green;
+
+    padding: 150px 0 !important;
+
+    @include mobile {
+      padding: $mobile-grid-padding !important;
+    }
+
+    p {
+      color: $white !important;
+    }
+
+    h1 {
+      color: $black;
+    }
+
+    h3 {
+      font-weight: $extra-bold;
+
+      @include tablet {
+        padding-bottom: 25px;
+      }
+
+      @include mobile {
+        padding-bottom: 0;
+      }
+    }
+
+    .prose {
+      font-weight: $medium;
```

Comment:
Maybe it's worth adding other details about font as well (like size, line height, letter spacing etc) for consistency?

Reply:
I didn’t specify other values, because they already match the design, cause these are the overrides and the only thing i wanted to change was font-weight